diffus/src/diffable_impls/ip.rs

```diff
@@ -35,13 +38,13 @@ macro_rules! ip_impl {
                         ($typ::V4(a), $typ::V4(b)) => match a.diff(&b) {
                             edit::Edit::Copy(_) => edit::Edit::Copy(self),
                             edit::Edit::Change(_) => {
-                                edit::Edit::Change(enm::Edit::AssociatedChanged((self, other)))
+                                edit::Edit::Change(enm::Edit::AssociatedChanged{before: self, after: other, diff: (self, other)})
```

Comment:
small formatting change

```suggestion
                                edit::Edit::Change(enm::Edit::AssociatedChanged { before: self, after: other, diff: (self, other) })
```